WooCommerce/Classes/POS/ViewModels/CartViewModel.swift

```diff
@@ -12,66 +11,31 @@ final class CartViewModel: CartViewModelProtocol {
     let addMoreToCartActionPublisher: AnyPublisher<Void, Never>
     private let addMoreToCartActionSubject: PassthroughSubject<Void, Never> = .init()
 
-    @Published private(set) var itemsInCart: [CartItem] = []
-    var itemsInCartPublisher: Published<[CartItem]>.Publisher { $itemsInCart }
+    var itemsInCartPublisher: AnyPublisher<[CartItem], Never> { posModel.$cart.eraseToAnyPublisher() }
 
     @Published var canDeleteItemsFromCart: Bool = true
-    @Published private(set) var shouldShowClearCartButton: Bool = false
```

Comment:
I'll keep it in mind and perhaps we can add it back at the end.... But for the time being I'm trying to get in the mindset of simply relying on the raw state we care about in the view as much as possible, without wrapping it up in extra functions and tests.

`posModel.cart.isNotEmpty && cartViewModel.canDeleteItemsFromCart` is already quite simple, and the `canDelete` bit is a wrapper around the current order stage (it needs to be `building`.) So I'm working towards it using those state items directly, and then we don't need to rely on combine flows to set flags, as we have here. 

That paves the way for us to try `@Observable` rather than `ObservableObject`, and has the potential to be more performant, as only changes to `orderStage` or `cart.isNotEmpty` would trigger this view being redrawn, rather than any change to `posModel` or `viewModel`, as happens with published properties on observable objects.